examples/rosa-classic-public-with-shared-vpc/main.tf

```diff
@@ -104,25 +108,34 @@ module "rosa_cluster_classic" {
 
   cluster_name                 = var.cluster_name
   operator_role_prefix         = module.operator_roles.operator_role_prefix
+  account_role_prefix          = module.account_iam_resources.account_role_prefix
   openshift_version            = var.openshift_version
-  replicas                     = var.replicas
-  installer_role_arn           = module.account_iam_resources.account_roles_arn["Installer"]
-  support_role_arn             = module.account_iam_resources.account_roles_arn["Support"]
-  controlplane_role_arn        = module.account_iam_resources.account_roles_arn["ControlPlane"]
-  worker_role_arn              = module.account_iam_resources.account_roles_arn["Worker"]
   oidc_config_id               = module.oidc_config_and_provider.oidc_config_id
   aws_subnet_ids               = module.shared-vpc-policy-and-hosted-zone.shared_subnets
-  multi_az                     = true
+  multi_az                     = length(module.vpc.availability_zones) > 1
+  replicas                     = length(module.vpc.availability_zones)
```

Comment:
Did that work? I'm under the impression that it needs 2 per availability zone :thinking:

Reply:
replicas is the number of worker machines, I'm not familiar with such limitation.
Maybe you confuse with subnets count, which require two subnets in availability zone (one public and one private)?